test/arm/neon/skel.c

```diff
@@ -19,7 +25,7 @@ test_simde_vxxx_f32 (SIMDE_MUNIT_TEST_ARGS) {
     simde_float32x2_t b = simde_vld1_f32(test_vec[i].b);
     simde_float32x2_t r = simde_vxxx_f32(a, b);
 
-    simde_test_arm_neon_assert_equal_f32x2(r, simde_vld1_f32(test_vec[i].r), 1);
+    simde_test_arm_neon_assert_equal_f32x2(r, simde_vld1_f32(test_vec[i].r));
```

Comment:
> Those aren't extra arguments; it's the precision for floating point tests (the number of decimal places we test to). They're only present on the floating point assertions, I'm guessing you thought they were extra because you were replacing them with integer assertions.
> 
> I'm going to drop these, but if I'm missing something please feel free to file another PR.

Sorry, I was missing something. I was using these for compare operations which, with float inputs, return an unsigned, not a float so I just changed e.g. f32 to u32 in the test without realizing that this was the reason for the extra argument error :-(